static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -162,9 +162,14 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
           ) : (
             <AuthStatus>
               {has2fa ? (
-                <IconCheckmark isCircled color="success" />
+                <IconCheckmark
+                  role="status"
+                  aria-label={t('2FA Enabled')}
+                  isCircled
+                  color="success"
+                />
```

Comment:
Honestly for testing you're better off just using `test-id`, since each icon has one. This should have `aria-hidden="true"` because as-is, going through the page it'll read "2FA Enabled, 2FA Enabled" since it'll read both the icon description _and_ the text after it. Just FYI! Same for the icon that follows. `"status"` seems reasonable here but that implies `aria-live="true"` which is pointless in this case, since the status doesn't update live anyway

Reply:
urgh, you're right about the text. Probably should just look for that on screen instead

Reply:
good call. Fixed up